apps/frontend/src/components/global/checklist/ChecklistRulesTable.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <v-card height="94vh" overflow-auto>
+    <v-card-title class="pt-2">
+      <div
+        style="
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          width: 100%;
+        "
+      >
+        <div>
+          <strong>
+            Rules ({{ numItems }} shown, {{ rules.length - numItems }} hidden)
+          </strong>
+        </div>
+        <div style="width: fit-content">
+          <v-switch
+            input-value="true"
+            style="margin-top: 0; padding-top: 0"
+            dense
+            inset
+            label="Short ID"
+            color="teal"
+            hide-details
+            @change="$emit('toggle-short-id')"
+          />
+        </div>
+      </div>
+      <v-spacer class="mt-0 pt-0" />
+      <v-select
+        v-model="selectedHeaders"
+        :items="headersList"
+        label="Select Columns"
+        class="mt-4 pt-0"
+        dark
+        item-color="white"
+        multiple
+        outlined
+        return-object
+        height="8.5vh"
+      >
+        <template #selection="{item, index}">
+          <div v-if="$vuetify.breakpoint.lgAndUp">
+            <v-chip v-if="index < 4" small>
```

Comment:
feels like there can be a lot of space left over to have more columns shown (i.e. these breakpoints aren't tight).  either we add in a lot more breakpoint handling or we just consolidate on having 3 or 4 regardless of size.

Reply:
Consolidated to just 3 for now